partner/plugins/index.php

```diff
@@ -55,11 +90,13 @@
                     </form>
                 </div>
                 <div class="modal-footer">
-                    <button type="button" class="btn btn-secondary closeModal">'.$lang["PARTNER"]["PLUGINS"]["CLOSE"].'</button>
+                    <button type="button" class="btn btn-secondary closeModal" data-bs-dismiss="modal">'.$lang["PARTNER"]["PLUGINS"]["CLOSE"].'</button>
                     <button type="button" class="btn btn-primary confirmPluginSettngs">'.$lang["PARTNER"]["PLUGINS"]["SAVE"].'</button>
```

Comment:
You have a breaking change. 

When confirmPluginSettngs is being clicked, it is posting the input fields multiple times.
The field was updated to "abcdefgh", which exist in the first webhook_url element, however in the second and third and fourth - it is the old inputted value.

![image](https://user-images.githubusercontent.com/5011158/175958832-aebfb21c-75cc-4718-8f7d-6ad12d5dee94.png)


The original sent payload:
![image](https://user-images.githubusercontent.com/5011158/175959355-1d1823a6-0244-4242-91af-563d6a719b5b.png)

Reply:
@MathiasGajhede Done, please review

Reply:
@MathiasGajhede  
1. Disable "Pay by link" for customers with the Company Channel "POS"
2. Change Partner Data to use form-switch instead of clean input-field
3. Plugins: Only show first 200 chars of description - open in Modal to show full text

https://github.com/IPPWorldwide/MerchantPortal/pull/2/commits/d35e9621d6199f655423d75b12390dbe04c468c0
This latest commit contains changes to all the above 3 listed tasks. When you will make a merge, it will reflect changes for the above 3 tasks.
If still have doubt about missing any code snippet, please review all commits of this branch. It will show all changes as my whole code/changes are present in this one "Pull Request".

For the next set of tasks, I am creating a new branch & making pull requests in the next few minutes to avoid any confusion.
Please review & share your feedback.